back-end/requirements.txt

```diff
@@ -0,0 +1,10 @@
+asgiref==3.5.1
+Django==4.0.4
+django-admin==2.0.1
+django-excel-base==1.0.4
+django-excel-response2==3.0.2
+django-six==1.0.4
+pytz==2022.1
+screen==1.0.1
+sqlparse==0.4.2
+xlwt==1.3.0
```

Comment:
Great start. Can we update the project's [Readme](https://github.com/Cosmic-Binturongs/slothstagram/blob/main/README.md) to include:

- What our project's back-end dependencies are
- How a user will, starting after `git clone`, have their back-end server running locally

Reply:
Hi! I updated the main readme with a section for installing dependencies for django

Reply:
I also edited README.md for virtual environments and proper dependencies installation with requirements.txt.